src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mlp_seq.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/mlp.hpp"
+
+#include "utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+namespace {
+
+std::vector<std::vector<InputShape>> inputShape_2D() {
+    auto shapes = SNIPPETS_TESTS_STATIC_SHAPES(
+        {{1, 64}},
+        {{2, 64}},
+        {{4, 64}},
+        {{8, 64}},
+        {{16, 64}},
+        {{32, 64}},
+        {{64, 64}});
+    return shapes;
+}
+
+std::vector<std::vector<InputShape>> inputShapeQuantized_2D() {
+    auto shapes = SNIPPETS_TESTS_STATIC_SHAPES(
+        {{1, 64}},
+        {{2, 64}},
+        {{4, 64}},
+        {{8, 64}},
+        {{16, 64}},
+        {{32, 64}},
+        {{64, 64}});
+    return shapes;
+}
```

Comment:
Do we really need to have two the same functions `inputShape_2D()` and `inputShapeQuantized_2D()`? Looks like we can reuse `inputShape_2D()` for quant tests

Reply:
Removed `inputShapeQuantized_2D` and reused `inputShape_2D`